test/e2e/antctl_test.go

```diff
@@ -33,6 +33,24 @@ type cmdAndReturnCode struct {
 	expectedReturnCode int
 }
 
+// TestAntctl is the top-level test which contains all subtests for
+// Antctl related test cases as they can share setup, teardown.
+func TestAntctl(t *testing.T) {
+	skipIfHasWindowsNodes(t)
+	skipIfNotRequired(t, "antctl")
+
+	data, err := setupTest(t)
+	if err != nil {
+		t.Fatalf("Error when setting up test: %v", err)
+	}
+	defer teardownTest(t, data)
+
+	t.Run("testAntctlAgentLocalAccess", func(t *testing.T) { testAntctlAgentLocalAccess(t, data) })
```

Comment:
Could you follow the code style in service_test.go? That looks neat and makes tests parallel.

Reply:
ah, I guess I misread your comment, I will reorg the code and add the parallel functions.